app/views/finance/provider/line_items/destroy.js.erb

```diff
@@ -1,7 +1,15 @@
-$('#<%= dom_id(@line_item) %>').remove();
+void function () {
+  <%# If EventTarget is replaceWith'ed immediately, ajax:success and ajax:complete are not called and the spinner will stay %>
+  setTimeout(() => {
+    document.getElementById('<%= dom_id(@line_item) %>').remove();
+  });
 
-$('#invoice_cost').html('<%= price_tag(@invoice.cost) %>');
-$('#invoice_cost_without_vat').html('<%= price_tag(@invoice.cost(vat_included: false)) %>');
+  <% unless @invoice.vat_rate.nil? %>
+    document.getElementById('invoice_cost_without_vat').innerHTML = '<%= price_tag(@invoice.cost(vat_included: false)) %>';
+  <% end %>
+
+  document.getElementById('invoice_cost').innerHTML = '<%= price_tag(@invoice.cost) %>';
```

Comment:
There is an issue that `invoice_cost_vat_amount` and the cell "Total Cost X Charged" which doesn't have an ID... but it is working like this in master, so - that's fine

Reply:
Yes, I created a JIRA to review this mess https://issues.redhat.com/browse/THREESCALE-11751